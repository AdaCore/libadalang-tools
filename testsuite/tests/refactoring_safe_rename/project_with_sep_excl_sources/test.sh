safe_rename -P default.gpr -S foo.ads -L 1 -R 10 -N Qux --algorithm=analyse_ast
