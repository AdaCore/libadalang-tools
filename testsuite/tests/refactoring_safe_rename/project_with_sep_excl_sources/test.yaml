description: |
    This test checks if the safe rename tool correctly handles
    projects with separate subprograms whose parent package source
    has been excluded from the project.

driver: shell_script
