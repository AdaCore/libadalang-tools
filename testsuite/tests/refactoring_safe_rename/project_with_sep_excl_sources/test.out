# Renaming node foo.ads:1:9: 
WARNING: File foo-bar-monitor.ads not found
WARNING: File foo-bar-monitor.adb not found
References:
Unit:foo-bar.ads
Line: 1
Source Location Range:
Start_Line: 1
Start_Column: 9
End_Line: 1
End_Column: 12
Line: 2
Source Location Range:
Start_Line: 2
Start_Column: 5
End_Line: 2
End_Column: 8
Unit:foo.ads
Line: 1
Source Location Range:
Start_Line: 1
Start_Column: 9
End_Line: 1
End_Column: 12
Line: 2
Source Location Range:
Start_Line: 2
Start_Column: 5
End_Line: 2
End_Column: 8
File renames:
foo-bar.ads -> qux-bar.ads
foo.ads -> qux.ads

